rotkehlchen/db/accounting_rules.py

```diff
@@ -117,30 +131,85 @@ def update_accounting_rule(
+                )
+            except sqlcipher.IntegrityError as e:  # pylint: disable=no-member
+                raise InputError('Overwritting rule already exists') from e
+
     def query_rules(
             self,
             filter_query: AccountingRulesFilterQuery,
-    ) -> tuple[
-        list[tuple[tuple[int, HistoryEventType, HistoryEventSubType, Optional[str]], BaseEventSettings]],  # noqa: E501
-        int,
-    ]:
+    ) -> tuple[list[RuleInformation], int]:
         """
         Query rules in the database using the provided filter. It returns the list of rules and
         the total amount of rules matching the filter without pagination.
         """
         query = 'SELECT * FROM accounting_rules '
         filter_query_str, bindings = filter_query.prepare()
-        rules = []
         with self.db.conn.read_ctx() as cursor:
             cursor.execute(query + filter_query_str, bindings)
-            rules = [
-                (entry[:4], BaseEventSettings.deserialize_from_db(entry[4:]))
+            rules = {
+                entry[0]: RuleInformation(
+                    identifier=entry[:4],
+                    rule=BaseEventSettings.deserialize_from_db(entry[4:]),
+                    overwrites={},
+                )
                 for entry in cursor
-            ]
+            }
             query, bindings = filter_query.prepare(with_pagination=False)
             query = 'SELECT COUNT(*) from accounting_rules ' + query
             total_found_result = cursor.execute(query, bindings).fetchone()[0]
-        return rules, total_found_result
+
+            # check the settings overwritting the rules
+            settings = self.db.get_settings(cursor)
+            cursor.execute('SELECT * FROM linked_rules_properties')
+            for (
+                _,
+                accountint_rule_id,
+                property_name,
+                setting_name,
+                is_active,
+            ) in cursor:
+                if bool(is_active) is False:
+                    continue
+
+                setting_value = getattr(settings, setting_name, None)
+                if setting_value is None:
+                    log.error(
+                        f'Failed to read setting value for {setting_name} in '
+                        'overwrites for accounting rules',
+                    )
+                    continue
+
+                if property_name == 'count_entire_amount_spend':
+                    old_value = rules[accountint_rule_id].rule.count_entire_amount_spend
+                    rules[accountint_rule_id].rule.count_entire_amount_spend = setting_value
+                elif property_name == 'count_cost_basis_pnl':
+                    old_value = rules[accountint_rule_id].rule.count_cost_basis_pnl
+                    rules[accountint_rule_id].rule.count_cost_basis_pnl = setting_value
+                else:
+                    log.error(f'Unknown accounting rule property {property_name}')
```

Comment:
Shouldn't you continue here?
```suggestion
                    log.error(f'Unknown accounting rule property {property_name}')
                    continue
```